packages/patternfly-4/content/design-guidelines/usage-and-behavior/lists-and-tables/design.md

```diff
@@ -91,6 +91,17 @@ Do not use for less structured or variably structured data that cannot be easily
 #### How to use
 Consider the structure of the data you want to display and organize that information into columns. Columns will typically have column headers. Every row within a table must have a consistent format. If the table row includes actions, they should always be placed in the rightmost column(s).
 
+### Sortable data table
+
+![sortable data table](img/sortable-data-table.png)
+
+1. **Sorted column**: when a column is being sorted by, the column header will turn blue and the sort icon will represent the direction of the sort. Subsequent clicks on the sortable column header will toggle the direction of the sort.
+2. **Hovered sort**: when a column is sortable, the sort icon will appear to the right of the column header. Upon hover, the  icon will change to a darker grey indicating that the icon is actionable.
+3. **Sortable column**: when a column is sortable, the sort icon will appear to the right of the column header in a light grey color. Sorting will not become active until the user selects the column header. This triggers the arrow to point upwards and the content to be sorted in ascending order.
+
+#### When to use
+Use a sortable data table over a standard data table. When column headers allow for sorting, it eases the ability to scan and read through the content. This is desired for data tables rather than adding sorting functionality to the toolbar.
```

Comment:
Use a sortable data table over a standard data table when?

Reply:
@mcarrano I was basically trying to say that we should always use a sortable table rather than a standard table since it is preferred to have sorting within the table rather than only in the toolbar. Would you agree?